api/jobspb/jobs.proto

```diff
@@ -0,0 +1,79 @@
+syntax = "proto3";
+
+package jobs;
+
+option go_package = "api/jobspb;jobspb";
+
+import "google/protobuf/duration.proto";
+import "google/protobuf/timestamp.proto";
+import "google/rpc/status.proto";
+
+// Ping is a AgentMessage/ServerMessage for checking connectivity, latency and clock drift.
+message Ping {}
+
+// Pong is an AgentMessage/ServerMessage with current time for measuring clock drift.
+message Pong {
+  google.protobuf.Timestamp timestamp = 1;
+}
+
+// StartJob asks pmm-agent to start job.
+message StartJob {
+  // Echo is a simple echo job that can be delayed.
+  message Echo {
+    string message = 1;
+    google.protobuf.Duration delay = 2;
+  }
+  string job_id = 1;
```

Comment:
this probably should generate agent not and not server

Reply:
Jobs API works as a bidirectional stream, so there is no explicit response where agent can notify server about what ID was assigned to the job. The server just throws StartJob message and waits for result/status/progress with the same ID.